src/elm/Main.elm

```diff
@@ -226,6 +229,20 @@ update msg model =
                             ]
                         )
 
+                GA.OpenEditTravelCard ->
+                    let
+                        accountModel =
+                            model.account
+
+                        newModel =
+                            { accountModel | editSection = Just AccountPage.TravelCardSection }
```

Comment:
Doesn't this belong to a function inside `Page.Account`? Then below, you'd have something like:

```elm
{ model | account = AccountPage.setEditSection AccountPage.TravelCardSection model.account }
```

Reply:
Enig. Usikker på kotyme med sider som eksporterer andre ting enn view/update/init. Men jeg sliter litt med eierforhold/struktur i Elm generelt. Mange ting som tvinges til å være spredt utover med unaturlig decoupling i mitt hode. Men å pakke det inn i et API gjør det i det minste litt bedre.